app/main.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+from . import models, schemas, crud
+from .db import SessionLocal, init_db
+from .utils import fetch_temperature
+
+app = FastAPI()
+
+# Инициализация БД
+init_db()
+
+# Получение сессии
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+# CRUD API для City
+@app.post("/cities/", response_model=schemas.City)
+def create_city(city: schemas.CityCreate, db: Session = Depends(get_db)):
+    return crud.create_city(db=db, city=city)

```

Comment:
Consider adding error handling for cases where the city creation might fail, such as when there are database constraints or validation errors.